corpus/structure.test

```diff
@@ -82,3 +82,20 @@ pipeline {
                           (declaration
                             name: (identifier)
                             value: (identifier)))))))))))))))
+==========
+Jenkins-import underscore
+==========
+@Library('my-library') _
+
+pipeline {
+}
+---
+(source_file
+  (declaration
+    (annotation
+      (identifier)
+      (argument_list
+        (string
+          (string_content)))))
+  (pipeline
+    (closure)))
```

Comment:
I haven't done the due diligence to understand the `@Library('my-library') _` syntax and what it's supposed to mean, so can't confirm this parse tree is good. But happy to merge if it works for you